CatAnalyzer/plugins/ttbbLepJetsAnalyzer.cc

```diff
@@ -867,7 +868,6 @@ void ttbbLepJetsAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetu
         b_Jet_partonFlavour->push_back(jet.partonFlavour());
         b_Jet_hadronFlavour->push_back(jet.hadronFlavour());
 
-<<<<<<< HEAD
 	// b-tag discriminant
 	float jet_btagDis_CSV = jet.bDiscriminator("pfCombinedInclusiveSecondaryVertexV2BJetTags");
```

Comment:
Could you change to
`float jet_btagDis_CSV = jet.bDiscriminator(BTAG_CSVv2);`
where you can find BTAG_CSVv2 literal in the Jet.